Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -437,6 +437,18 @@
 ### MinBias fastsim_13 TeV for mixing, 2018 ###                                                                                                                                      
 workflows[2018.8] = ['',['MinBiasFS_13_UP18_ForMixing']]
```

Comment:
Hi @ssekmen , you also need to define all the modules used in these noPU workflows, such as HARVESTUP18FS, MINIAODMCUP18FS, TTbarFS_13_UP18, SingleMuPt10FS_UP18, .... 
You can add them in relval_step.py similar as what you did for 2017 in PR #23363, such as 
https://github.com/cms-sw/cmssw/pull/23363/files#diff-819af2ec677272fddf1fef50e788a88bR925
https://github.com/cms-sw/cmssw/pull/23363/files#diff-819af2ec677272fddf1fef50e788a88bR2278
https://github.com/cms-sw/cmssw/pull/23363/files#diff-819af2ec677272fddf1fef50e788a88bR2394

Reply:
This PR should be a combination of #23363 (noPU) and #24829 (PU and premix). But now you missed all the module definitions in #23363, which will also be used in #24829, for example, you added this line: 
https://github.com/cms-sw/cmssw/blob/ddae68805e360eaa743039a8e18ffb42d7399646/Configuration/PyReleaseValidation/python/relval_steps.py#L1062 
But TTbarFS_13_UP18was not defined

Reply:
Hi @zhenhu , thanks for the tips.  I have made the additions, however when I try to test a workflow, e.g. 2018.1 , I keep getting the error "overwritting FS_ZEE_13_PRMXUP18_PU50 not allowed".  I see FS_ZEE_13_PRMXUP18_PU50 occuring only in one place, in the premix wf 250400.18 . I don't see what that has to do with a standard wf.  I still pushed my changes to help diagnose.